drag_and_drop_v2/templates/html/js_templates.html

```diff
@@ -9,80 +9,84 @@
     </div>
 </script>
 
-<script id="zone-input-tpl" type="text/html">
+<script class="zone-input-tpl" type="text/html">
     <div class="zone-row" data-uid="{{zone.uid}}">
-        <!-- uid values from old versions of the block may contain spaces and other characters, so we use 'index' as an alternate unique ID here. -->
-        <label for="zone-{{index}}-title">{{i18n "Text"}}</label>
-        <input type="text"
-               id="zone-{{index}}-title"
-               class="title"
-               value="{{ zone.title }}"
-               required />
-        <a href="#" class="remove-zone hidden">
+        <a class="remove-zone hidden" title="{{i18n 'Remove zone'}}">
             <div class="icon remove"></div>
         </a>
-        <label for="zone-{{index}}-description">{{i18n "Description"}}</label>
-        <input type="text"
-               id="zone-{{index}}-description"
-               class="description"
-               value="{{ zone.description }}"
-               placeholder="{{i18n 'Describe this zone to non-visual users'}}"
-               required />
-        <div class="layout">
-            <label for="zone-{{index}}-width">{{i18n "width"}}</label>
+        <label>
+            <span>{{i18n "Text"}}</span>
             <input type="text"
-                   id="zone-{{index}}-width"
-                   class="size width"
-                   value="{{ zone.width }}" />
-            <label for="zone-{{index}}-height">{{i18n "height"}}</label>
+                   class="title"
```

Comment:
This class name (`title`) is too generic; Studio already defines a `[class*="view-"] .modal-window .title` rule which is affecting this. I'd suggest `zone-title` instead.